src/components/View.js

```diff
@@ -28,6 +28,7 @@ function View({
   const goBack = useCallback(() => pop(), [pop]);
 
   const showBackButton = size > 1 && !!pop;
+  const backIsLogout = size === 2;
```

Comment:
should probably be a constant instead of magic variable since it relies on assumptions about our router depth